### DISCUSSION TITLE
Improvement of `pikaur -G`

### DISCUSSION THREAD

**Discussion by Bundy01** [actionless/pikaur#715] [Ideas]
Hi,

It would be nice if this command could display the PKGBUILD directly in the terminal.
The downloaded folder should be in `/tmp/pikaur` rather than $HOME.

You could add an option in `~/.config/pikaur.conf` for users who would prefer to keep the downloaded folder in $HOME.

Regards.

**Comment by actionless** [comment]
it's clear about download location (actually it defaults to current working directory, ie PWD, not HOME)

but i didn't understood the usecase/scenario regarding displaying it

**Comment by Bundy01** [comment]
I'm thinking to show PKGBUILD directly to `stdout`.

**Comment by Bundy01** [comment]
@actionless > What do you think of the idea of being able to consult the PKGBUILDs of uninstalled packages directly in the terminal?

**Reply by actionless** [reply]
i hardly could imagine a common usecase scenario for such feature, could you give some examples?

**Comment by Bundy01** [comment]
This allows you to consult the PKGBUILD of an uninstalled package directly. Currently, if I want to do this, I have to go to the Archlinux site, search for the package and then read this file...

**Reply by actionless** [reply]
this i understood from your previous message, but what's the scenario/usecase?

**Comment by Bundy01** [comment]
For example, to know the dependencies, to know if the file is well written before starting the compilation. This avoids typing the required password with `pikaur -S`.

**Reply by actionless** [reply]
but pikaur asking you to review the pkgbuild before the compilation anyway, so i still can't understand the scenario itself

**Comment by Bundy01** [comment]
Currently, you have to type your password to access to the file. This is the only way to access it (or I don't know the method).

**Reply by actionless** [reply]
yeah, but you'll need to type it anyway sooner or later for installation

**Comment by Bundy01** [comment]
Yes, if I install it, but in this case, it would just be to consult it, if the file suits me, I'll install it.
Otherwise, I don't. That would avoid the password.

What about the idea of putting folders in /tmp/pikaur?

**Reply by actionless** [reply]
yeah it could be useful to make those dir paths (https://github.com/actionless/pikaur#file-locations) configurable via pikaur.conf

**Reply by actionless** [reply]
or you mean the location of file downloaded by -G command?

**Reply by Bundy01** [reply]
For the -G command, but if you think of other situations?
